src/components/localization/localized-link.tsx

```diff
@@ -82,25 +82,7 @@ export const SharedLinkStyle = css<SharedLinkStyleProps>`
         `}
 `
 
-const ShareDisabledStyle = css<{ disabled: boolean }>`
-    ${({ disabled }) =>
-        disabled &&
-        `
-        pointer-events: none;
-        opacity: 0.32;`}
-`
-
-const StyledAnchor = styled.a`
-    ${ShareDisabledStyle}
-`
-
-const StyledAnchorLink = styled(AnchorLink)`
-    ${ShareDisabledStyle}
-`
-
-const StyledGatsbyLink = styled(GatsbyLink)`
-    ${ShareDisabledStyle}
-`
+const StyledGatsbyLink = styled(GatsbyLink)``
```

Comment:
Just `<GatsbyLink>....</GatsbyLink>`

Reply:
@niloo-fs no, `GatsbyLink` can't accept styles, so we use `StyledGatsbyLink`, I deleted all styled components where it was possible. Example: `<StyledGatsbyLink aria-label={aria_label}` , `aria-label={aria_label}` its a style props and `GatsbyLink ` can't accept it 👍

Reply:
Alrighty @eduard-deriv , thank u for your explanations